cosmwasm/devnet/config/genesis.json

```diff
@@ -403,7 +403,7 @@
     },
     "staking": {
       "params": {
-        "unbonding_time": "600s",
+        "unbonding_time": "1814400s",
```

Comment:
Same here.

Reply:
This is also required to sync terra2 and wormchain using IBC. An `unbonding_time` of 10 minutes translates into a trusting period equal to 0, which isn't allowed. `unbonding_time = 1814400s` is the default value which translates to a valid trusting period.

Reply:
Ack.  Please document this either in this config file or in the commit message.